pkg/events.go

```diff
@@ -438,3 +470,56 @@ func (octopus *EventOctopus) SaveOrUpdate(event Event) error {
 	}
 	return tx.Commit().Error
 }
+
+// recover creates a map from event.Uuid to event.Name
+// for all items in the map that do not have the event.Name == EventCompleted, a new event will be published
+// unless the max retry count has been reached.
+func (octopus *EventOctopus) recover() error {
+	var events []Event
+
+	if err := octopus.Db.Debug().Order("uuid").Find(&events).Error; err != nil {
```

Comment:
Why order by UUID and not time of just no ordering?

Reply:
probably from an old mindset where I though there would be more than 1 event per uuid (eg only inserts) but since events are updated, it should only be one. So this can be removed.